suites/luminous/ansible/switch-from-non-containerized-to-containerized-ceph-daemons.yaml

```diff
@@ -0,0 +1,45 @@
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-83571467
+      module: test_ansible.py
+      config:
+        ansi_config:
+            ceph_test: True
+            ceph_origin: distro
+            ceph_stable_release: luminous
+            ceph_repository: rhcs
+            osd_scenario: collocated
+            osd_auto_discovery: False
+            journal_size: 1024
+            ceph_stable: True
+            ceph_stable_rh_storage: True
+            fetch_directory: ~/fetch
+            copy_admin_key: true
+            ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_default_pool_size: 2
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
+            cephfs_pools:
+              - name: "cephfs_data"
+                pgs: "8"
+              - name: "cephfs_metadata"
+                pgs: "8"
+      desc: osd with collocated journal
+      destroy-cluster: False
+      abort-on-fail: true
+
+   - test:
+      name: switch-from-non-containerized-to-containerized-ceph-daemons
+      polarion-id: CEPH-83573510
+      module: switch-from-non-containerized-to-containerized-ceph-daemons.py
```

Comment:
Suggesting you to update with shorter module name.

Reply:
@sunilkumarn417 The playbook name used by ceph-ansible is ```switch-from-non-containerized-to-containerized-ceph-daemons.yml" hence I have used it. However, I can change it to ```rpm_to_container.py``` with the addition of a doc string as suggested earlier?